src/assets/scripts/layout/theme.js

```diff
@@ -12,11 +12,12 @@ import $ from 'jquery';
 import {pageLinkFocus} from '@shopify/theme-a11y';
 import {cookiesEnabled} from '@shopify/theme-cart';
 import {wrapTable, wrapIframe} from '@shopify/theme-rte';
+import sections from '@shopify/theme-sections';
 
 window.slate = window.slate || {};
 window.theme = window.theme || {};
 
-/* = =============== Templates ================ */
+window.theme.sections = sections;
```

Comment:
Maybe we don't need this now since we assign the same thing to `window.__sections__` in `@shopify/theme-sections`?